gitpod-io/gitpod
# Show a pending status if setup is still required

## Description

For single org mode installations this shows a pending setup status on the `<Login/>` view if setup is still underway (no sso configured yet).

| With Welcome | Without Welcome |
|--------|--------|
| ![image](https://github.com/gitpod-io/gitpod/assets/367275/cbe62bb4-5044-4a0c-bb15-ccc24306b2f0) | ![image](https://github.com/gitpod-io/gitpod/assets/367275/d8d4de2e-1d97-44c3-b79b-22e1a59fc11b) | 

## Related Issue(s)

Fixes WEB-300

## How to test

* Visiting the preview environment will show this state currently, as I haven't setup SSO on it yet. 
* Once SSO has been setup for an environment, this pending UI should swap to the normal Login view w/ a Continue button.

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - bmh-setup-ef9fed0a87</li>
	<li><b>🔗 URL</b> - <a href="https://bmh-setup-ef9fed0a87.preview.gitpod-dev.com/workspaces" target="_blank">bmh-setup-ef9fed0a87.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options:

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold
